packages/docs/src/pages/en/features/dates.md

```diff
@@ -338,3 +338,35 @@ export interface DateAdapter<TDate> {
   getNextMonth (date: TDate): TDate
 }
 ```
+
+## Inheritance
+
+You can also extend and override build-in DateAdapter using class inheritance:
+
+```ts
+import { VuetifyDateAdapter } from 'vuetify/lib/composables/date'
```

Comment:
Never import anything from /lib. This should be added to the exports map in package.json, probably `vuetify/date/adapters/vuetify` (same pattern as locale adapters) to go with #21174.

Reply:
👍🏼  thank for being vigilant